OPAL/ll/src/test/scala/org/opalj/ll/fpcf/analyses/ifds/taint/NativeBackwardTaintAnalysis.scala

```diff
@@ -24,7 +24,8 @@ import org.opalj.ll.llvm.value.Call
 import org.opalj.tac.fpcf.properties.Taint
 
 /**
- * This is a simple IFDS based backward taint analysis
+ * This is a simple IFDS based backward taint analysis.
+ * This is a mostly a demo, especially since the source/sink/sanitize methods are not configurable.
```

Comment:
Can/should we move it to a more suitable project/package then? Or does this have to stay here?